lib/ui/contact_list/widgets/chat_list_item_tile.dart

```diff
@@ -70,34 +69,44 @@ class ChatListItemTile extends ConsumerWidget {
     GroupType groupType,
   ) {
     final groupsNotifier = ref.watch(groupsProvider.notifier);
+    final fallbackName = groupsNotifier.getGroupDisplayName(group.mlsGroupId) ?? group.name;
```

Comment:
[sug] maybe the logic of display name for DMs could be moved to groups notifier?  But this was a late comment and don't wan't to stop you from merging,  just a suggestion.